Explorer/Assets/DCL/Input/UnityEventSystem.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using UnityEngine.EventSystems;
+
+namespace DCL.Input
+{
+    public class UnityEventSystem : IEventSystem
```

Comment:
The only reason I used the `EventSystem` was to avoid creating a mother canvas with a `GraphicsRaycaster`, so I'll just rename this to `UIRaycaster`, if we use more of the `EventSystem` in the future we can refactor the name